tests/ui/undocumented_unsafe_blocks.rs

```diff
@@ -490,4 +490,20 @@ unsafe impl CrateRoot for () {}
 // SAFETY: ok
 unsafe impl CrateRoot for (i32) {}
 
+fn issue_9142() {
+    // SAFETY: ok
+    let _ = unsafe {};
```

Comment:
Can you change this to be on two lines. e.g.
```rust
// SAFETY: ok
let _ =
    unsafe {}
```

Reply:
Ah, rustfmt pulled this together.